spec/lucky_web/form_helpers_spec.cr

```diff
@@ -46,19 +46,19 @@ end
 describe LuckyWeb::FormHelpers do
   it "renders a form tag" do
     view.inferred_put_form.to_s.should contain <<-HTML
-    <form action="/form_helpers/fake_id" method="post"><input type="hidden" name="_method" value="put"/> foo </form>
+    <form action="/form_helpers/fake_id" method="post"><input type="hidden" name="_method" value="put"/> foo</form>
```

Comment:
Did you run `crystal spec`? I think this may be broken because of the space before `foo`. (I need to get CI setup lol)

Reply:
haha, makes sense. 👍 to CI